src/app/cases/page.tsx

```diff
@@ -48,64 +57,67 @@ export default function Page() {
+          defaultValue={defaultFilterValues.remote}
+          multi
+          options={new Set(['Remote', 'In Person'])}
+          value={caseFilters.remote}
+          onChange={v => setCaseFilters({ ...caseFilters, remote: v })}
         />
         <FilterDropdown
-          defaultValue="Attorney/Interpreter"
-          options={['Attorney/Interpreter', 'Interpreter Only']}
-          onChange={v => setCaseFilters({ ...caseFilters, role: v as string })}
+          defaultValue={defaultFilterValues.role}
+          multi
+          options={new Set(['Interpreter', 'Attorney'])}
+          value={caseFilters.role}
+          onChange={v => setCaseFilters({ ...caseFilters, role: v })}
         />
+        {/* <FilterDropdown
+          defaultValue={defaultFilterValues.languages}
+          multi
+          options={new Set(caseData.flatMap(c => c.languages))}
+          value={caseFilters.languages}
+          onChange={v => setCaseFilters({ ...caseFilters, languages: v })}
+        /> */}
         <FilterDropdown
-          defaultValue="Anywhere"
-          // requires knowing user location
-          options={[
-            'Within 5 miles',
-            'Within 10 miles',
-            'Within 20 miles',
-            'Within 100 miles',
-            'Anywhere',
-          ]}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, distance: v as string })
-          }
+          defaultValue={defaultFilterValues.agency}
+          multi
+          options={new Set(['Court', 'USCIS'])}
+          value={caseFilters.agency}
+          onChange={v => setCaseFilters({ ...caseFilters, agency: v })}
         />
         <FilterDropdown
-          defaultValue="All countries"
+          defaultValue={defaultFilterValues.countries}
           multi
-          options={[
-            'All countries',
-            ...Array.from(
-              new Set(
-                caseData
-                  .filter(c => c.country)
-                  .map(c => (c.country ? c.country : '')),
-              ),
-            ),
-          ]}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, countries: v as string[] })
+          options={
+            new Set(
+              caseData.filter(c => c.country).map(c => c.country),
+            ) as Set<string>
           }
+          value={caseFilters.countries}
+          onChange={v => setCaseFilters({ ...caseFilters, countries: v })}
         />
       </FiltersContainer>
       <MainDisplay>
         <CardColumn>
           {caseData
             .filter(c => {
-              if (caseFilters.remote === 'remote') return c.is_remote;
-              if (caseFilters.remote === 'inperson') return !c.is_remote;
-              return true;
+              if (caseFilters.remote.size === 0) return true;
+              if (caseFilters.remote.has('Remote') && c.is_remote) return true;
+              if (caseFilters.remote.has('In Person') && !c.is_remote)
+                return true;
```

Comment:
nit: can we condense this somehow? maybe

```
c => caseFilters.remote.size === 0 || (caseFilters.remote.has('Remote') && c.is_remote) || (caseFilters.remote.has('In Person') && !c.is_remote)
```

Reply:
ye, thanks for flagging - forgot about these filters tbh